docs/services/vision/segmentation.md

```diff
@@ -218,6 +219,80 @@ The following parameters are available for a `detector_3d_segmenter`.
 
 Click **Save config** and proceed to [test your segmenter](#test-your-segmenter).
 
+## Configure an `obstacles_depth` segmenter
+
+This segmenter model is for depth cameras, and is best for motion planning with transient obstacles.
+
+Use the segmenter to identify well separated objects above a flat plane.
+If you want to identify well separated point clouds over the flat plane, set `with_geometries: true`.
+[Configure your frame system](/services/frame-system/#configuration) to configure the relative spatial orientation of the components of your robot within Viam's [frame system service](/services/frame-system/).
+After configuring your frame system, your camera will populate its own `Properties` with the intrinsic parameters from the frame system, as long you have configured the parameters as part of your camera.
+You can then get those parameters from your camera through the [camera API](/components/camera/#getproperties).
+The segmenter now returns point clouds within the `GeometryInFrame` object it captures.
+
+If you choose not to configure the frame system to provide instrinsic parameters, set `with_geometries: false`.
```

Comment:
@npentrel Added your suggestions here and above, I think! Let me know if it still doesn't make sense and we can maybe sync up next week to get this right?

Reply:
I think I'm still not clear what I am identifying when `with_geometries: false`. On lines 208/209, we say the segmenter identifies well separated objects above a flat plane, and if we want then we should set `with_geometries: true`. But am I still identifying well separated objects when I have that set to `false`? Is the difference just that I don't get the coordinates of where those are? 

As a reader I still don't fully understand why I'd choose one over the other. An example may help here.